controls/stig_rhel9.yml

```diff
@@ -1847,7 +1847,8 @@ controls:
             - medium
         title: RHEL 9 SSH daemon must be configured to use system-wide crypto policies.
         rules:
-            - configure_ssh_crypto_policy
+            - harden_sshd_ciphers_opensshserver_conf_crypto_policy
```

Comment:
I am not sure if this rule should be here. Reading the relevant STIG, it seems that the audit part does not mention desired ciphers at all. But the rule you use actually checks for exact ciphers in files provided by crypto-policies package. I think we need a new rule as this STIG checks rather for correct include directives than for actual ciphers configured.

Reply:
New rule added.